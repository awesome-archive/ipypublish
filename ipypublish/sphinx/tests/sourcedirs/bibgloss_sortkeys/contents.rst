Welcome to IpySphinx extension tests documentation!
===================================================


:glsc:`gtkey1`, :gls:`akey1`, :gls:`symbol1`

.. rubric:: Glossary

.. bibglossary:: test_glossary.bib



