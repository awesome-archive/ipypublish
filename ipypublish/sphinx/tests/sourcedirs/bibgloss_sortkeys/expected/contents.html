
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Welcome to IpySphinx extension tests documentation! &#8212; Python  documentation</title>
    <link rel="stylesheet" href="_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <script type="text/javascript" src="_static/language_data.js"></script>
    <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <div class="section" id="welcome-to-ipysphinx-extension-tests-documentation">
<h1>Welcome to IpySphinx extension tests documentation!<a class="headerlink" href="#welcome-to-ipysphinx-extension-tests-documentation" title="Permalink to this headline">Â¶</a></h1>
<p><a class="bibglossary bibgcapital reference internal" href="#gtkey1" id="id1">Name</a>, <a class="bibglossary reference internal" href="#akey1" id="id2">MA</a>, <a class="bibglossary reference internal" href="#symbol1" id="id3"><span class="math notranslate nohighlight">\(\pi\)</span></a></p>
<p class="rubric">Glossary</p>
<p id="bibtex-bibglossary-contents-0"><table class="bibglossary docutils citation" frame="void" id="gtkey1" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id1">[name]</a></td><td>full description <strong>with latex</strong></td></tr>
</tbody>
</table>
<table class="bibglossary docutils citation" frame="void" id="symbol1" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id3">[<span class="math notranslate nohighlight">\(\pi\)</span>]</a></td><td>full description</td></tr>
</tbody>
</table>
<table class="bibglossary docutils citation" frame="void" id="akey1" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id2">[MA]</a></td><td>My Abbreviation</td></tr>
</tbody>
</table>
</p>
</div>


          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="#">Python</a></h1>








<h3>Navigation</h3>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="#">Documentation overview</a><ul>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3>Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" />
      <input type="submit" value="Go" />
      <input type="hidden" name="check_keywords" value="yes" />
      <input type="hidden" name="area" value="default" />
    </form>
    </div>
</div>
<script type="text/javascript">$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 1.8.5</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.10</a>
      
      |
      <a href="_sources/contents.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>